fs/path/mod.ts

```diff
@@ -1,1431 +1,25 @@
-    // after any path separator we find
-    let preDotState = 0;
-
-    // Get non-dir info
-    for (; i >= start; --i) {
-      const code = path.charCodeAt(i);
-      if (code === CHAR_FORWARD_SLASH) {
-        // If we reached a path separator that was not part of a set of path
-        // separators at the end of the string, stop now
-        if (!matchedSlash) {
-          startPart = i + 1;
-          break;
-        }
-        continue;
-      }
-      if (end === -1) {
-        // We saw the first non-path separator, mark this as the end of our
-        // extension
-        matchedSlash = false;
-        end = i + 1;
-      }
-      if (code === CHAR_DOT) {
-        // If this is our first dot, mark it as the start of our extension
-        if (startDot === -1) startDot = i;
-        else if (preDotState !== 1) preDotState = 1;
-      } else if (startDot !== -1) {
-        // We saw a non-dot and non-path separator before our dot, so we should
-        // have a good chance at having a non-empty extension
-        preDotState = -1;
-      }
-    }
-
-    if (
-      startDot === -1 ||
-      end === -1 ||
-      // We saw a non-dot character immediately before the dot
-      preDotState === 0 ||
-      // The (right-most) trimmed path component is exactly '..'
-      (preDotState === 1 && startDot === end - 1 && startDot === startPart + 1)
-    ) {
-      if (end !== -1) {
-        if (startPart === 0 && isAbsolute) {
-          ret.base = ret.name = path.slice(1, end);
-        } else {
-          ret.base = ret.name = path.slice(startPart, end);
-        }
-      }
-    } else {
-      if (startPart === 0 && isAbsolute) {
-        ret.name = path.slice(1, startDot);
-        ret.base = path.slice(1, end);
-      } else {
-        ret.name = path.slice(startPart, startDot);
-        ret.base = path.slice(startPart, end);
-      }
-      ret.ext = path.slice(startDot, end);
-    }
-
-    if (startPart > 0) ret.dir = path.slice(0, startPart - 1);
-    else if (isAbsolute) ret.dir = "/";
-
-    return ret;
-  },
-
-  sep: "/",
-  delimiter: ":",
-  win32: null,
-  posix: null
-};
-
-posix.win32 = win32.win32 = win32;
-posix.posix = win32.posix = posix;
-
-const module = isWindows ? win32 : posix;
-
-export const resolve = module.resolve;
-export const normalize = module.normalize;
-export const isAbsolute = module.isAbsolute;
-export const join = module.join;
-export const relative = module.relative;
-export const toNamespacedPath = module.toNamespacedPath;
-export const dirname = module.dirname;
-export const basename = module.basename;
-export const extname = module.extname;
-export const format = module.format;
-export const parse = module.parse;
-export const sep = module.sep;
-export const delimiter = module.delimiter;
+import * as _win32 from "./win32.ts";
+import * as _posix from "./posix.ts";
+
+import { isWindows } from "./constants.ts";
+
+const path = isWindows ? _win32 : _posix;
```

Comment:
Here rename `module` variable to `path`.

Just in case of support for the `commonjs` module in the future